omero/sysadmins/system-migration.rst

```diff
@@ -0,0 +1,127 @@
+#. Once both have been installed, perform an OMERO.server `Backup and restore`_ procedure.
+
+
+Choosing a platform
+-------------------
+
+The two recommended platforms are RHEL/RockyLinux 9 and Ubuntu 22.04.
+
+Installation
+------------
+
+The installation walkthroughs provided in the documentation try to stick to a minimum installation.
+Check the main :doc:`server <unix/server-installation>` and :doc:`web <unix/install-web/web-deployment>` installation pages.
+
+However, more advanced installation mechanisms are available if you are interested and have familiarity
+with the given mechanism:
+
+- `Ansible roles <https://galaxy.ansible.com/ui/standalone/namespaces/5249/>`_ are available for most installation steps. 
+  The primary roles, `omero-server <https://galaxy.ansible.com/ui/standalone/roles/ome/omero_server/>`_ and `omero-web <https://galaxy.ansible.com/ui/standalone/roles/ome/omero_web/>`_ allow to install respectively the OMERO.server and OMERO.web.
+
+- `Docker images <https://hub.docker.com/u/openmicroscopy>`_ are also available. Both the `omero-server`
+  and `omero-web` images are considered production quality.
+
+Please get in touch at https://forum.image.sc/c/data if you have any questions.
+
+
+Backup and restore
+------------------
+
+Mininally, you will need to back up and restore the binary repository and the database.
+
+
+Backup
+~~~~~~
+
+- Perform the backups following the :doc:`backup instructions<server-backup-and-restore>`. 
+- When ready, copy them to the new operating system::
+
+    rsync -ra --progress db.dump NEW_OPERATING_SYSTEM:/tmp
+    rsync -ra --progress OMERO_BINARY_BACKUP NEW_OPERATING_SYSTEM:/tmp
+
+
+On the new operating system, stop the OMERO.server.
+
+Restore the binary
+~~~~~~~~~~~~~~~~~~
+
+- Remove all the directories under ``/OMERO`` e.g. ``sudo rm -rf /OMERO/ManagedRepository``.
+
+- Change ownership and group of the renamed directories under ``OMERO_BINARY_BACKUP`` (exception is the backup diretory containing the ``omero.config`` i.e. ``/tmp/OMERO_BINARY_BACKUP/backup``)::
+
+    sudo chown -R omero-server /tmp/OMERO_BINARY_BACKUP/ManagedRepository/
+    sudo chgrp -R managed_repo_group /tmp/OMERO_BINARY_BACKUP/ManagedRepository/
+
+- As the ``omero-server`` user, move the renamed backup directories under ``/tmp/OMERO_BINARY_BACKUP`` (exception ``/tmp/OMERO_BINARY_BACKUP/backup``) to ``/OMERO``, for example::
+
+    sudo -u omero-server -s
+    mv /tmp/OMERO_BINARY_BACKUP/ManagedRepository /OMERO/ManagedRepository
+ 
+
+Restore the database
+~~~~~~~~~~~~~~~~~~~~
+
+- Restore the database backup following the :doc:`restore PostgresQL instructions<server-backup-and-restore>`.
+
+Restore the OMERO.server configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Restore the backup configuration::
+
+   omero config load /tmp/OMERO_BINARY_BACKUP/backup/omero.config
+
+You should then follow the *Reconfiguration* steps of
+:doc:`install <unix/server-installation>`.
```

Comment:
this is too imprecise, I genuinely do now know what Reconfiguration steps are meant - the doc which is linked is not having any Reconfiguration section, except for a section which looks like the one here - it feels very circular.